main/res/values/preference_keys.xml

```diff
@@ -82,11 +82,8 @@
     <string translatable="false" name="pref_supersizeDistance">supersizeDistanceToggle</string>
     <string translatable="false" name="pref_plainLogs">plainLogs</string>
     <string translatable="false" name="pref_signature">signature</string>
-    <string translatable="false" name="pref_logTemplate1">pref_logTemplate1</string>
-    <string translatable="false" name="pref_logTemplate2">pref_logTemplate2</string>
-    <string translatable="false" name="pref_logTemplate3">pref_logTemplate3</string>
-    <string translatable="false" name="pref_logTemplate4">pref_logTemplate4</string>
-    <string translatable="false" name="pref_logTemplate5">pref_logTemplate5</string>
+    <string translatable="false" name="pref_logTemplate_category">pref_category_logTemplate</string>
```

Comment:
Can you remove the "pref_" in the string? "pref_" should only be part of the string's name, not of its content.

Reply:
Only for the category or also for the templates itself? For the LogTemplates I had kept the `pref_` for migration purposes, but this feature was only visible in the nightlys for just a couple of days anyway...

Reply:
Whatever, I will simply change it as migrating will not work anyway when using the counter variable...